core/modules/replica/Configuration.h

```diff
@@ -472,6 +550,11 @@ class Configuration {
     static std::string  const defaultDatabaseUser;
     static std::string  const defaultDatabasePassword;
     static std::string  const defaultDatabaseName;
+    static size_t       const defaultDatabaseServicesPoolSize;
+    static bool               defaultDatabaseAllowReconnect;        // read-write
+    static unsigned int       defaultDatabaseConnectTimeoutSec;     // read-write
+    static unsigned int       defaultDatabaseMaxReconnects;         // read-write
+    static unsigned int       defaultDatabaseTransactionTimeoutSec; // read-write
```

Comment:
by my reading of https://developer.lsst.io/cpp/style.html#private-class-variables-and-methods-must-have-underscore-prefix these are all supposed to be prefixed with an underscore. There is ambiguity because these are protected and the doc refers to private, and does not seem to take a stance on protected. But my reading of the justification makes me think that protected is the same as private for this situation.

Reply:
oh, except here https://developer.lsst.io/cpp/style.html#the-parts-of-a-class-must-be-sorted-public-protected-and-private
the doc does not prefix the protected var:
```
...
protected:
    float aFloat;
    float doSomethingElse();

private:
    char _aChar;
...
```
so it would seem that I'm wrong.

Reply:
Noted